src/components/DatePicker/DatePicker.jsx

```diff
@@ -156,10 +159,10 @@ export const DatePicker = ({
         );
     };
 
-    const rangeModifier = isRangeVariant ? { start: value.from, end: value.to } : null;
+    const rangeModifier = isRangeVariant ? { start: value?.from, end: value.to } : null;
```

Comment:
what is the difference in the values now ?  i see everywhere now we are using optional chaining ?

Reply:
Yep, I removed unnecessary optional changes. It was needeed only for end month